### DISCUSSION TITLE
Elements must be unique inside his block?

### DISCUSSION THREAD

**Discussion by Tecayehuatl** [getbem/getbem.github.io#257] [Q&A]
Some authors say we don't have to have elements with the same name within his block, they say those must be treated like an ID, which means not to have the same name, but I have seen too many cases like this (They use the "navigation-bar-list__list-item" class many times):

```
<header class="main-header">
	<nav class="navigation-bar">
		<ul class="navigation-bar-list">
			<li class="navigation-bar-list__list-item"></li>
			<li class="navigation-bar-list__list-item"></li>
			<li class="navigation-bar-list__list-item"></li>
		</ul>
	</nav>
</header>

```

^How do you handle this?

I thinks that BEM tree is right, but I don't know if is valid to don't to fall in DRY in CSS properties (in the below case for no repeating CSS properties):

```
<header class="main-header">
	<nav class="navigation-bar">
		<ul class="navigation-bar-list">
			<li class="navigation-bar-list__list-item-1"></li>
			<li class="navigation-bar-list__list-item-2"></li>
			<li class="navigation-bar-list__list-item-3"></li>
		</ul>
	</nav>
</header>
```

**Comment by Tecayehuatl** [comment]
According this video: [video](https://youtu.be/IO-4Z32O--c?t=3m18s) "Elements must be unique inside his block"

**Comment by tadatuta** [comment]
There's just no reason for treating elements as IDs. You may use as many elements with the same name as you wish.
